webapi/src/main/twirl/org/knora/webapi/messages/twirl/queries/sparql/admin/getFileValueStandard.scala.txt

```diff
@@ -50,6 +50,10 @@ CONSTRUCT {
         knora-base:attachedToProject ?resourceProject .
 
     ?fileValue ?objPred ?objObj .
+
+    @* This FILTER is unnecessary, but it makes Jena run the query faster. *@
```

Comment:
that comment means that we know how the triplestore works ;-)

Reply:
Exactly! :)